valheim_plus.cfg

```diff
@@ -366,4 +366,106 @@ enabled=false
+; Damage Modifier: Normal; Resistant; Weak; Immune; Ignore; VeryResistant; VeryWeak
+; Example:
+; eikthyrBuffDamageTypesModifiers = "Fire:Immune|Pierce:VeryWeak" ; This will make the Eikthyr buff immune to fire and very weak to pierce attacks
+; Description: Text you want to customize for the buff; note if empty it takes the default one.
+restedBuffHealthRegenMultiplier = 50
+restedBuffStaminaRegenMultiplier = 100
+restedBuffJumpStaminaModifier = 0
+restedBuffRunStaminaModifier = 0
+
+restingBuffHealthRegenMultiplier = 100
+restingBuffStaminaRegenMultiplier = 200
+
+warmBuffBuffHealthRegenMultiplier = 0
+warmBuffBuffStaminaRegenMultiplier = 100
+
+corpseRunBuffHealthRegenMultiplier = 0
+corpseRunBuffStaminaRegenMultiplier = 0
+corpseRunBuffJumpStaminaModifier = -75
+corpseRunBuffRunStaminaModifier = -75
+
+freezingDebuffHealthRegenMultiplier = 0
+freezingDebuffStaminaRegenMultiplier = -60
+freezingDebuffDamagePerTick = -1
+
+coldDebuffHealthRegenMultiplier = -50
+coldDebuffStaminaRegenMultiplier = -25
+
+wetDebuffHealthRegenMultiplier = -25
+wetDebuffStaminaRegenMultiplier = -15
+wetDebuffDuration = 120
+
+eikthyrBuffHealthRegenMultiplier = 0
+eikthyrBuffStaminaRegenMultiplier = 0
+eikthyrBuffJumpStaminaModifier = -60
+eikthyrBuffRunStaminaModifier = -60
+eikthyrBuffCooldown = 1200
+eikthyrBuffDuration = 300
+eikthyrBuffModifyAttackSkill = 
+eikthyrBuffDamageModifier = 0
+eikthyrBuffDamageTypesModifiers = 
+eikthyrBuffDescription = 
+
+theElderBuffHealthRegenMultiplier = 0
+theElderBuffStaminaRegenMultiplier = 0
+theElderBuffJumpStaminaModifier = 0
+theElderBuffRunStaminaModifier = 0
+theElderBuffCooldown = 1200
+theElderBuffDuration = 300
+theElderBuffModifyAttackSkill = WoodCutting
+theElderBuffDamageModifier = 60
+theElderBuffDamageTypesModifiers = 
+theElderBuffDescription = 
+
+bonemassBuffHealthRegenMultiplier = 0
+bonemassBuffStaminaRegenMultiplier = 0
+bonemassBuffJumpStaminaModifier = 0
+bonemassBuffRunStaminaModifier = 0
+bonemassBuffCooldown = 1200
+bonemassBuffDuration = 300
+bonemassBuffModifyAttackSkill = 
+bonemassBuffDamageModifier = 0
+bonemassBuffDamageTypesModifiers = Blunt:Resistant|Slash:Resistant|Pierce:Resistant
+bonemassBuffDescription = 
+
+yagluthBuffHealthRegenMultiplier = 0
+yagluthBuffStaminaRegenMultiplier = 0
+yagluthBuffJumpStaminaModifier = 0
+yagluthBuffRunStaminaModifier = 0
+yagluthBuffCooldown = 1200
+yagluthBuffDuration = 300
+yagluthBuffModifyAttackSkill = Fire:Resistant|Frost:Resistant|Lightning:Resistant
+yagluthBuffDamageModifier = 0
+yagluthBuffDamageTypesModifiers = 
+yagluthBuffDescription = 
+
+moderBuffHealthRegenMultiplier = 0
+moderBuffStaminaRegenMultiplier = 0
+moderBuffJumpStaminaModifier = 0
+moderBuffRunStaminaModifier = 0
+moderBuffCooldown = 1200
+moderBuffDuration = 300
+moderBuffModifyAttackSkill = 
+moderBuffDamageModifier = 0
+moderBuffDamageTypesModifiers = 
+moderBuffDescription = 
+
```

Comment:
This is bloating config file WAY too much

too many configuration items

Reply:
It can be separated into a dedicated plugin, but then it will allow players to have their own config not sync with the server.
This is the reason why I did that PR to have it inside V+ which handles that but I totally understand your points ;-).

I don't know if you have any plan to have sub-modules or any approach that would basically allow/reject other plugins on the connection to the server and allow their configuration exchanges.

Let me know your thought on that.

Reply:
That is something to look into

part of the reason we have trouble with external plugins is being ale to enable/set them server side

best idea is likely to for all mods to implement simialr solution to ours where you get the settings from server, but this is more of a game limitation sadly

Reply:
If you want me to simplify the effects list and settings, let me know.
I can have some time today to clean it and adjust the one you guys wants.